python/pyspark/ml/classification.py

```diff
@@ -1959,6 +1950,15 @@ def setParams(self, featuresCol="features", labelCol="label", predictionCol="pre
         kwargs = self._input_kwargs
         return self._set(**kwargs)
 
+    @since("2.0.0")
```

Comment:
@huaxingao keeping it at 2.0.0 is okay ?

Reply:
@dilipbiswal 
Yes, it's ok to keep 2.0.0. The method was originally in 2.0.0 in the subclass of `OneVsRest`. Now it was moved to `OneVsRest`.

Reply:
@huaxingao OK..